sklearn/decomposition/truncated_svd.py

```diff
@@ -71,26 +71,33 @@ class TruncatedSVD(BaseEstimator, TransformerMixin):
     ----------
     components_ : array, shape (n_components, n_features)
 
-    explained_variance_ratio_ : array, [n_components]
-        Percentage of variance explained by each of the selected components.
-
     explained_variance_ : array, [n_components]
         The variance of the training samples transformed by a projection to
         each component.
 
+    explained_variance_ratio_ : array, [n_components]
+        Percentage of variance explained by each of the selected components.
+
+    singular_values_ : array, [n_components]
+        The singular values corresponsing to each of the selected components.
+        The singular values corresponds to the 2-norms of the ``n_components``
```

Comment:
Maybe say ``are the 2-norms`` or ``are equal to``. You just used "corresponding". Also, there's a typo in "corresponding".